verifiers/internal/gha/slsaprovenance/v1.0/base.go

```diff
@@ -186,3 +193,12 @@ func (p *provenanceV1) GetSystemParameters() (map[string]any, error) {
 
 	return sysParams, nil
 }
+
+// GetExternalParameters() implements Provenance.GetExternalParameters.
```

Comment:
It's not clear that this method needs to be made public outside of the package? Can it just be `getExternalParameters()`?